extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -290,7 +290,7 @@ public function execute() {
 							"city_dbname"
 						),
 						($this->mCityID) ? array( "city_list.city_id" => $this->mCityID ) : array( "city_list.city_dbname" => $this->mCityDB ),
-						__METHOD__
+						__METHOD__ . '::domaindb'
```

Comment:
I guess the "db" suffix doesn't hold any meaning here as well as in 2 places below. Can we get rid of it?